### PR TITLE
Improve code comparison section

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/data/home.tsx [hunk 1]
```diff
@@ -273,21 +273,13 @@ export default {
...
-            <b className={typographyStyles.note}>⚠ Note:</b> Both Formik and
-            Redux-Form snippets below were copied from the official
-            documentation.
-          </p>
-        </>
+        <p id="codeComparison">
+          Reducing the amount of code that you have to write is one of the
+          primary goals of React Hook Form. To illustrate that, let's look at
+          this following code example.
+        </p>
```
Comment (pmaier983) [inline src/data/home.tsx:282]:
```suggestion
        <p id="codeComparison">
          Reducing the amount of code you need to write, and removing
          unnecessary re-renders are some of the primary goals of React Hook
          Form. Now dive in and explore with the following example:
        </p>
```

Comment (pmaier983) [reply src/data/home.tsx:282]: I tested this locally and it causes the `example:` to be on its own in the final line, so I suggest the following change as well:

// container.module.css (Lines 24-27)
```
.centerContent p {
  max-width: 720px;
  margin: 0 auto 10px;
}
```

**Review by pmaier983 — APPROVED** [review]
I think the CodeSandbox is a great value add! Look awesome!

**Comment by bluebill1049** [comment]
@pmaier983 thanks and feel free to introduce the css update as well :) then we can merge and go live. U should have that CSB access as well

**Comment by bluebill1049** [comment]
thanks a lot, @pmaier983 deploying 🚀  I will close the other PR as well.

**Pull request merged**